.goreleaser.yml

```diff
@@ -0,0 +1,68 @@
+# Goreleaser configuration
+# for building binaries and packages for distributions and releasing on github
+dist: bin
+
+git:
+  short_hash: true
+
+builds:
+  - binary: semaphore
+    main: ./cli/main.go
+    goos:
+      - windows
+      - darwin
+      - linux
+      - freebsd
+      - openbsd
+      - netbsd
+    goarch:
+      - amd64
+      - 386
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: 386
+    hooks:
+      pre: task compile
+
+archive:
+  format_overrides:
+    - goos: windows
+      format: zip
+  files:
```

Comment:
Should we put some kind of README (and config.yaml sample) here?

Reply:
well the readme that is currently in the root kind of says nothing, which is why i didnt include it, its not really very helpful if you want to learn about semaphore. Maybe that is a more general improvement that we could make though.

You can get a config sample by running `semaphore -config` right? The validity of that approach is another question ;) Also you should run setup so you get the database user created, and I feel like adding a config file suggests that if you use it everything will 'just work' which is not the case, and as a new user finding that the missing piece is that you didnt do setup and the user was not created is not actually that easy to find out

Reply:
This is the only one I havn't made a change for yet. What do you think we should do?

Reply:
We may put it to separate task, focused on providing of documentation.